client/my-sites/checkout/checkout-thank-you/hundred-year-thank-you/index.tsx

```diff
@@ -189,37 +209,48 @@ export default function HundredYearThankYou( {
 	const hundredYearDomainCta = (
 		<StyledLightButton
 			onClick={ () =>
-				page( ` /domains/manage/all/${ registeredDomain.name }/edit/${ registeredDomain.name }` )
+				page( ` /domains/manage/all/${ targetDomain?.name }/edit/${ targetDomain?.name }` )
 			}
 		>
 			{ translate( 'Manage your domain' ) }
 		</StyledLightButton>
 	);
 	const cta = productSlug === PLAN_100_YEARS ? hundredYearPlanCta : hundredYearDomainCta;
 
+	const messageTarget = targetDomain?.domain || siteSlug;
+	const domainSpecificDescription =
+		productSlug === domainProductSlugs.DOTCOM_DOMAIN_REGISTRATION
+			? translate( 'Your 100-Year Domain %(messageTarget)s has been registered.', {
+					args: {
+						messageTarget,
+					},
+			  } )
+			: translate( 'Your 100-Year Domain %(messageTarget)s is being transferred.', {
+					args: {
+						messageTarget,
+					},
+			  } );
```

Comment:
```suggestion
	const domainSpecificDescription =
		productSlug === domainProductSlugs.DOTCOM_DOMAIN_REGISTRATION
			? translate( 'Your 100-Year Domain %(domain)s has been registered.', {
					args: {
						domain: targetDomain?.domain || siteSlug, // though targetDomain?.domain should be defined here, right?
					},
			  } )
			: translate( 'Your 100-Year Domain %(domain)s is being transferred.', {
					args: {
						domain: targetDomain?.domain, || siteSlug,
					},
			  } );
```
Small suggestion, I'm not sure if translators can see the placeholder names and/or use them to aid in their translations